vassal-doc/src/main/readme-referencemanual/ReferenceManual/MassPieceLoader.adoc

```diff
@@ -72,6 +72,10 @@ The options you have to choose from are:
 * _Same as:_
 * _Use Base image:_
 
+==== Version 3.5.5 limitations
```

Comment:
@claudiociardelli This limitation isn't version specific is it? This has always and will always be the case?

Reply:
Removed ref to version.